prodavnica/src/main/java/com/levi9/prodavnica/serviceImpl/BookServiceImpl.java

```diff
@@ -45,7 +45,8 @@ public BookListDTO findAllBooks() {
 				.collect(Collectors.toList());
```

Comment:
Maybe include the condition 'isDeleted' in a database query, so it will only fetch active books, so there is no need to iterate through them